### DISCUSSION TITLE
Discuss the resource index layout pattern

### DISCUSSION THREAD

**Discussion by johanstromqvist** [Shopify/polaris-react#8215] [Pattern documentation]
Welcome to the resource index layout pattern discussion!

### Here are some things you can do here:

💡 **Share ideas** on how to make this pattern, or its documentation, better.

❓ **Ask a specific question** about this pattern. 

---
Tips before posting:
- Remember Polaris and our discussions are public. If you work for Shopify and have an externally sensitive topic, please reach out in #polaris in Slack
- Please [report bugs here](https://github.com/Shopify/polaris/issues/new?assignees=&labels=%F0%9F%90%9BBug&template=ISSUE.md)
- [Contributions](https://polaris.shopify.com/contributing) are always welcome
- Learn more about [Polaris discussions](https://github.com/Shopify/polaris/discussions/6750)

💜👩‍🚀✌️ the Polaris team

**Comment by johanstromqvist** [comment]
We might want to link to Empty states in Related resources when we have a pattern page ready for that.

**Comment by hitesh-kc** [comment]
We are not abled to select product on mobile device on index table filter, is there any solution to select index row on small screen?

**Reply by johanstromqvist** [reply]
Hi @hitesh-kc! Could you please elaborate on what the problem is? Is this a problem with the component, the layout pattern, or the functionality on the pattern documentation page? Thanks!

**Reply by hitesh-kc** [reply]
So is the problem with the index filter component, that we are not able to select a row on mobile screen

**Reply by johanstromqvist** [reply]
Ah, I see. Thanks!

Would you mind creating a [component bug report](https://github.com/Shopify/polaris/issues/new?assignees=&labels=%F0%9F%90%9BBug&template=ISSUE.md), please?

**Reply by chloerice** [reply]
Hey @hitesh-kc 👋🏽 The IndexTable React component intentionally does not support bulk action at mobile breakpoints. The experience is best on the native iOS/Android Shopify app when using a mobile device.

**Comment by ChielvanderVlist** [comment]
I am curious as to how you would distinguish the index table from the source list component. On a UX level, they seem very similar in purpose. So how do you decide to use the index table or the source list?
